### DISCUSSION TITLE
Barrel roll spinner elements

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#15849] [Ideas]
so as im sure you are all aware of the spinner text when playing BR is not locked in  place making it spin with everything else,.

i have not been able to prove empirically this empirically but i suspect the spnner itself also isnt  having any sort of counter balance applied when accounting for number of times spun, meaning spinning at 300rpm probably results in a lower spin rate when going with BRs spin direction and  a higher one when going against it, 

Considering this I am willing to attempt to modify BR to not interact with spinners, would this behaviour be desirable?

**Comment by peppy** [comment]
I'm not sure why this would be wanted.

**Reply by mk56-spn** [reply]
![osu_2021-11-29_09-17-28](https://user-images.githubusercontent.com/74463310/143831613-c4f77d09-eee8-4291-b65d-3f47179ed2a7.jpg)
idk i dont feel like this is good for playability or aesthetics

**Reply by integerrr** [reply]
I dont see an issue with playability as any player with more than 5 minutes of experience at the game knows how a spinner works. Aesthetics is a moot point with barrel roll cuz intuitively players already expects stuff in the playfield to spin with the mod
